src/compiler/cpp_generator.h

```diff
@@ -45,86 +45,86 @@ namespace grpc_cpp_generator {
 // Contains all the parameters that are parsed from the command line.
```

Comment:
Note you will need to change/remove line 32 to 41.

Reply:
You're right. Thanks!